src/components/Map/SelectionOverlay.jsx

```diff
@@ -1,10 +1,40 @@
 import {Box, Chip} from "@mui/material";
+import * as MuiIcons from '@mui/icons-material';
 
-export const SelectionOverlay = (props)=>{
+export const SelectionOverlay = (props) => {
   const {chips, location, chat} = props;
+
   const onClick = (chip) => {
-    chat({ messages:  [{ role: "user", content: chip }], location });
+    chat({messages: [{role: "user", content: chip.description}], location});
```

Comment:
Shouldn't this be async if using the mutation

Reply:
it just dispatches an action, no need for async / .then